svg.js.d.ts

```diff
@@ -429,8 +434,14 @@ declare namespace svgjs {
         f: number;
         matrix: Matrix;
     }
+
+    interface Library {
+        Matrix: Matrix;
+    }
+
     export interface Matrix {
         (source: any): Matrix;
+        new(source: any): Matrix;

```

Comment:
this makes sense though i think it highlights an even bigger problem with the typings.

when i wrote the typings i forgot to supply the arguments to the methods on the library
what i mean is, this works
```typescript
var matrix: svgjs.Matrix = new SVG.Matrix();
```
but this dose not
```typescript
var element: svgjs.Element = new SVG.Element();
var matrix: svgjs.Matrix = new SVG.Matrix(element);
```
this is a mistake on my end since i set the `Matrix` as a function on the `Library` and not a property that holds a reference to the `Matrix` class
a few examples, [here](https://github.com/svgdotjs/svg.js/blob/4770c1e3b817f0ded7403f35edb4f724bf36af2c/svg.js.d.ts#L461) and [here](https://github.com/svgdotjs/svg.js/blob/4770c1e3b817f0ded7403f35edb4f724bf36af2c/svg.js.d.ts#L366)
but i remembered to include in other places, like [here](https://github.com/svgdotjs/svg.js/blob/4770c1e3b817f0ded7403f35edb4f724bf36af2c/svg.js.d.ts#L366)

of course this means that we would have to run over the whole library and fix the parts like this

```typescript
interface Library { Number(): void; }
```
and change then to something like this
```typescript
interface Library { Number: Number; }
```
and then fix this
```typescript
export interface Number{
    // change this
    (value: any, unit?: any): Number;
    // to this, that way SVG.Number can only be called with the "new"
    new (value: any, unit?: any): Number;
}
```
so while this dose fix it for the Matrix class i think we will end up having to go over the entire file and fix each class